packages/curve-ui-kit/src/features/switch-chain/ui/ChainList.tsx

```diff
@@ -10,6 +9,7 @@ import TextField from '@mui/material/TextField'
 import Typography from '@mui/material/Typography'
 import { t } from '@ui-kit/lib/i18n'
 import { CheckedIcon } from '@ui-kit/shared/icons/CheckedIcon'
+import { SearchIcon } from '@ui-kit/shared/icons/SearchIcon'
```

Comment:
Why not use the new SearchField component here as well?

Reply:
Sure